deploy/templates/notificationtemplates/userdeactivated/notification.html

```diff
@@ -50,6 +50,11 @@
         You can request new access by signing up again at {{.RegistrationURL}}
     </p>
 
+    <p>
+        Share your feedback, request an extension for your Sandbox environment from the #dev-sandbox channel on DevNation Slack.
+        You can join using the following <a href="https://dn.dev/DevNationSlack">invitation</a>.
```

Comment:
We previously had issues with rendering links in our emails. We need to test if it works now.